### ISSUE TITLE
Angr currently failing to import on arm based macos.  

### ISSUE THREAD

**Issue by social-anthrax** [angr/angr#4133]
### Description

There's currently a bug in capstone where the installed universal wheel for mac is only for intel:
https://github.com/capstone-engine/capstone/issues/2154. 

~~Mac users can go back to angr version 9.2.65 to avoid this error.~~
Mac users can install angr using `pip install angr --no-binary capstone`, or if using poetry reinstall the venv after setting `poetry config --local installer.no-binary capstone`. 

The issue is present in the 5.0.0.post1, 5.0.0-rc4, 5.0.0-rc2 capstone builds.

### Steps to reproduce the bug

On arm based MacOS:
pip install angr==9.2.67
python3 -c "import angr"

### Environment

python3 -m angr.misc.bug_report causes an error.
Output of `poetry show --tree` below, poetry.lock attached. 
```
angr 9.2.67 A multi-architecture binary analysis toolkit, with the ability to perform dynamic symbolic execution and various static analyses on binaries
├── ailment 9.2.67
├── archinfo 9.2.67
├── cachetools *
├── capstone 5.0.0.post1
├── cffi >=1.14.0
│   └── pycparser *
├── claripy 9.2.67
│   ├── cachetools *
│   ├── decorator *
│   ├── pysmt >=0.9.5
│   └── z3-solver 4.10.2.0
├── cle 9.2.67
│   ├── pefile *
│   ├── pyelftools >=0.27
│   ├── pyvex 9.2.67
│   │   ├── archinfo 9.2.67
│   │   ├── bitstring *
│   │   │   └── bitarray >=2.8.0,<3.0.0
│   │   └── cffi >=1.0.3
│   │       └── pycparser *
│   └── sortedcontainers >=2.0
├── colorama *
├── cppheaderparser *
│   └── ply *
├── dpkt *
├── gitpython *
│   └── gitdb >=4.0.1,<5
│       └── smmap >=3.0.1,<6
├── itanium-demangler *
├── mulpyplexer *
├── nampa *
│   └── future *
├── networkx >=2.0,<2.8.1 || >2.8.1
├── protobuf >=3.19.0
├── psutil *
├── pycparser >=2.18
├── pyvex 9.2.67
│   ├── archinfo 9.2.67
│   ├── bitstring *
│   │   └── bitarray >=2.8.0,<3.0.0
│   └── cffi >=1.0.3
│       └── pycparser *
├── rich >=13.1.0
│   ├── markdown-it-py >=2.2.0
│   │   └── mdurl >=0.1,<1.0
│   └── pygments >=2.13.0,<3.0.0
├── rpyc *
│   └── plumbum *
│       └── pywin32 *
├── sortedcontainers *
├── sympy *
│   └── mpmath >=0.19
└── unicorn 2.0.1.post1
```
[poetry.lock.txt](https://github.com/angr/angr/files/12540858/poetry.lock.txt)

### Additional context

No action expected from angr devs as this is an upstream bug.

**Comment by mahaloz** [comment]
I encountered this error a few weeks ago (https://github.com/angr/angr-management/issues/1029). If you are willing to fix this yourself, you can clone down capstone and build, then `pip install -e` to the same env you have angr in.

**Comment by social-anthrax** [comment]
@mahaloz I just checked up on this, there are also some alternatives without having to clone the full repo and also has the benefit of being more easily updatable later. 
`pip install angr --no-binary capstone`. 

If you use poetry then you can reinstall your venv after setting 
`poetry config --local installer.no-binary capstone` and it should work.

**Comment by mahaloz** [comment]
@twizmwazin any chance a real solution for this is happening soon? Or are we still stalled by capstone-side devs?

**Comment by twizmwazin** [comment]
This is fixed as of capstone 5.0.3
